transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    url='https://sites.astro.caltech.edu/ztf/bts/explorer.php?f=s&amp;subsample=trans&amp;classstring=&amp;classexclude=&amp;ztflink=lasair&amp;lastdet=&amp;startsavedate=&amp;startpeakdate=&amp;startra=&amp;startdec=&amp;startz=&amp;startdur=&amp;startrise=&amp;startfade=&amp;startpeakmag=&amp;startabsmag=&amp;starthostabs=&amp;starthostcol=&amp;startb=&amp;startav=&amp;endsavedate=&amp;endpeakdate=&amp;endra=&amp;enddec=&amp;endz=&amp;enddur=&amp;endrise=&amp;endfade=&amp;endpeakmag=19.0&amp;endabsmag=&amp;endhostabs=&amp;endhostcol=&amp;endb=&amp;endav=&amp;reverse=y&amp;format=csv'
</pre><p> </p><p>We discussed on 13 July: just be explicit that the "ZTF BTSS explorer" allows us to make this selection and generate this list of targets and that should be sufficient explanation.</p><p> </p><p> </p><p>[Not to make life harder, but I wonder if they have an API...]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>